test/e2e/tests/ppom/ppom-blockaid-alert-erc20-transfer.spec.js

```diff
@@ -6,57 +6,163 @@ const {
   unlockWallet,
   withFixtures,
 } = require('../../helpers');
-const { SECURITY_ALERTS_PROD_API_BASE_URL } = require('./constants');
+const { mockSecurityAlertsAPIFailed } = require('./utils');
 const { mockServerJsonRpc } = require('./mocks/mock-server-json-rpc');
 
-const SELECTED_ADDRESS = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
+const selectedAddress = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
+const selectedAddressWithoutPrefix = '5cfe73b6021e818b776b421b1c4db2474086a7e1';
 
-const CONTRACT_ADDRESS_USDC = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48';
+const CONTRACT_ADDRESS = {
+  BalanceChecker: '0xb1f8e55c7f64d203c1400b9d8555d050f94adf39',
+  FiatTokenV2_1: '0xa2327a938febf5fec13bacfb16ae10ecbc4cbdcf',
+  OffChainOracle: '0x52cbe0f49ccdd4dc6e9c13bab024eabd2842045b',
+  USDC: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+};
 
 async function mockInfura(mockServer) {
+  await mockSecurityAlertsAPIFailed(mockServer);
+
   await mockServerJsonRpc(mockServer, [
```

Comment:
As this is a lot of test data, is it worth a dedicated `ppom-mocks.ts` in this directory with functions for each mock?